src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int height;
+    private int width;
+    private String color = getColor();
```

Comment:
There is no need to initialize the 'color' field here since it's being set in the constructor. You can remove the initialization here.